src/scripts/webpack/build-ts.js

```diff
@@ -0,0 +1,14 @@
+import webpackBuild from './webpack.build.js';
+import compiler from './compiler';
+import { logger } from '../../util';
+import { statsOutputConfiguration } from './util';
+import replaceBabelWithTypescript from './replaceBabelWithTypescript';
+
+const getBuildConfig = (...args) => replaceBabelWithTypescript(
```

Comment:
I wanted to avoid passing stuff to webpack and then to Webpack.base.. But I'll try as it's probably more clear in the end